tests/integration/test_upgrade.py

```diff
@@ -46,7 +46,7 @@ def test_category_not_sorted_without_directive(pipenv_instance_private_pypi):
         assert c.returncode == 0
         assert list(p.pipfile["packages"].keys()) == [
             "zipp",
-            "six",
             "colorama",
             "atomicwrites",
+            "six",
```

Comment:
This should be acceptable because its the default non-sorting case; it is technically a behavioral change that re-installing something moves it to the bottom of the Pipfile, but I feel we get closed to reducing some complexity around the Pipfile creation, and writing in this PR -- it doesn't seem to negatively impact performance either.